website/docs/cdktf/feature-flags.mdx

```diff
@@ -0,0 +1,39 @@
+---
+page_title: Feature Flags
```

Comment:
Hi! I think we've already got some of this content, so we don't need a new page for it :-). 

We talk about it here: https://www.terraform.io/cdktf/release#enable-features. and then we also have a corresponding section with more detail in the actual config file page: https://www.terraform.io/cdktf/create-and-deploy/configuration-file#enable-features

Would you want to make edits to that content on the config file page instead of adding this new page?